lib/experimentUtils.ts

```diff
@@ -34,17 +34,6 @@ export const MESSAGING_EXPERIMENTS_DEFAULT_FEATURES : string[] = [
   "spotlight",
```

Comment:
Adding "testFeature" here will keep the tests happy, since we're filtering experiments now :)